packages/replay/src/replay.ts

```diff
@@ -807,7 +820,7 @@ export class ReplayContainer implements ReplayContainerInterface {
       // This means we retried 3 times and all of them failed,
       // or we ran into a problem we don't want to retry, like rate limiting.
       // In this case, we want to completely stop the replay - otherwise, we may get inconsistent segments
-      this.stop('sendReplay');
+      void this.stop('sendReplay');
```

Comment:
Just noticed this @mydea -- I think this is ok since `stop()` will attempt the flush, fail, and when it retries, `this._isEnabled` is `false`, so will fail.

Reply:
I think it should be fine since `this._isEnabled = false` should be set "synchronously" in the async stop function. So I would expect this to be false already in the next code line, basically.